src/python/serenity_types/pricing/derivatives/rates/yield_curve.py

```diff
@@ -142,6 +142,16 @@ class YieldCurveDefinition(BaseModel):
     The curve's intended purpose, e.g. for discounting or representing market view on forward rates.
     """
 
+    rate_source_types: List[RateSourceType]
+    """
+    For reference purposes, the list of rate sources that were used to bootstrap this curve.
+    """
+
+    interpolation_method: InterpolationMethod
+    """
+    The specific interpolator type used to bootstrap this curve.
+    """
```

Comment:
@kdowney-cloudwallcapital Should we move this to YieldCurveDefinition instead?
And from the code, seems like rate_source_types will always be just one value i.e. not a list, if that's so I can add the fix

Reply:
No, in the most general case it is a list; we may piece together the curve from more than one source in the future.

Reply:
And I am fine with this change to shift the metadata entirely to YieldCurveDefinition, it makes sense to me.